capstone/src/components/User/Experience/RegisterExperience/RegisterExperience.jsx

```diff
@@ -75,28 +94,37 @@ export default function RegisterExperience() {
       </Heading>
       {fields.map((inputField) => (
         <FormControl isRequired={inputField.isRequired} key={inputField.id} p="20px">
-          <FormLabel>{inputField.displayText}</FormLabel>
-          {inputField.input === 'input'
-            ? (
-              <Input
-                value={inputField.value}
-                type={inputField.type}
-                placeholder={inputField.placeholder}
-                onChange={(e) => handleRegistryChange(inputField.id, e.target.value)}
-              />
-            )
-            : (
-              <Textarea
-                value={inputField.value}
-                type={inputField.type}
-                placeholder={inputField.placeholder}
-                onChange={(e) => handleRegistryChange(inputField.id, e.target.value)}
-              />
-            )}
+          <FormLabel>
+            {inputField.displayText}
+          </FormLabel>
+          {/* QUESTION: How to refactor that code? */}
```

Comment:
Hmm, as much as it's great to avoid repetition, this might be a place where the repetition helps readability? Not sure